src/main/java/seedu/mtracker/instrument/subinstrument/Forex.java

```diff
@@ -1,15 +1,34 @@
 package seedu.mtracker.instrument.subinstrument;
 
 import seedu.mtracker.instrument.Instrument;
+import seedu.mtracker.ui.TextUi;
 
 public class Forex extends Instrument {
+    protected double entry;
+    protected double exit;
+    protected String expiry;
+    protected String remarks;
 
-    public Forex(String name, double currentPrice, String sentiment) {
+    private static final String FX_ICON = "F";
+
+    public Forex(
+            String name,
+            double currentPrice,
+            String sentiment,
+            double entryPrice,
+            double exitPrice,
+            String deadline,

```

Comment:
Consider name it to expiry to keep naming consistent

Reply:
ok